src/core/core.controller.js

```diff
@@ -923,7 +924,7 @@ module.exports = function(Chart) {
 			}
 
 			// Built in hover styling
-			if (me.active.length && hoverOptions.mode) {
+			if (me.active.length && hoverOptions.mode && helpers.valueOrDefault(hoverOptions.enabled, true)) {
```

Comment:
I don't think we need these extra check because if `me.active.length > 0`, that means hover is enabled.

Also, I can't find any reference to `hover.enabled`, is it a new option?